src/proxyClient.js

```diff
@@ -97,7 +97,14 @@ var frameId = 0;
 
 // Worker
 
-var worker = new Worker('{{{ filename }}}.js');
+var filename = '{{{ filename }}}.js';
+var workerURL = filename;
+var fileBytes = tryParseAsDataURI(filename);
+if (fileBytes) {
+  workerURL = URL.createObjectURL(new Blob([fileBytes], {type: 'application/javascript'}));
+}
+
+var worker = new Worker(workerURL);
```

Comment:
Any updates on this? If deprecating `MEM_INIT_METHOD=2` is agreed on, investigating runtime size optimization could be left as one of the follow-on tasks.

Alternatively, I could remove the deprecation commit and submit it as a separate PR for greater visibility and/or dedicated discussion among all interested parties.